src/python/genny/parsers/cedar.py

```diff
@@ -22,6 +22,7 @@
 from os.path import join as pjoin
 
 from bson import BSON
+from bson.int64 import Int64
```

Comment:
FWIW, https://github.com/mongodb/mongo-python-driver/blob/3.7.2/bson/__init__.py#L89 should mean you could just write `bson.Int64` instead of needing to import it separately.

Reply:
GTK.. Although I don't know how anyone could discover that :P

The documentation uses the long import path: https://api.mongodb.com/python/current/api/bson/index.html